### ISSUE TITLE
Roadmap to v1.0

### ISSUE THREAD

**Issue by AlexisRenchon** [CliMA/ClimaLand.jl#684]
**Remaining issues to close by mid September at the latest:**
- [x] PR #1387, will close #1205  and #655 @AlexisRenchon 
- [x] https://github.com/CliMA/ClimaLand.jl/issues/1412 @juliasloan25 
- [x] https://github.com/CliMA/ClimaLand.jl/issues/1414 @AlexisRenchon and add biomass information
- [x] Move LAI and rooting depth out of PlantHydraulics - see #600 @kmdeck #1388 
- [x] C4 Pmodel #1329 #1356 @kmdeck  
- [x] Broken doc links #256 #1345 @AlexisRenchon 
- [x] constructors not appearing in docs #1346  @AlexisRenchon 
- [x]  nearest neighbor interp - regenerate IC #1296 @kmdeck 
- [x] automated required model callbacks #1381 @kmdeck 
- [x] vg params by depth #1304 #1303 @kmdeck 
- [x] Verify time offsets for fluxnet sites #1343 @juliasloan25 
- [x] default diagnostics can depend on model parameterizations #1305 @juliasloan25 
- [x] unify callbacks #1316 @imreddyTeja 
- [x] pmodel parameters in toml file https://github.com/CliMA/ClimaLand.jl/issues/1374 @ph-kev 
- [x] #1309 @imreddyTeja 
- [x] #608 - All - plots for experiments in the paper
- [x] add note that soil co2 is in beta @kmdeck 
- [x] remove unused constructors @kmdeck https://github.com/CliMA/ClimaLand.jl/pull/1408

**Nice to have:**
- [ ] After #1330 merges, create a planthydraulics option "SteadyState" which does not solve for \vartheta - no prognostic variables, and computes the root water and energy flux differently (using transpiration) @kmdeck  
- [ ] remove dt from snow model #1352 @kmdeck 
- [ ] Nans in soil CO2 #842 #1072 #992  @kmdeck 
- [x] close #1186 
- [ ] #1169 #1373 @kmdeck 
- [x] #1272 @juliasloan25 
- [x] #1270   closed by #1385 @ph-kev 
- [ ] #657  - ? - easy fix for scientific improvement
- [ ] #674  -@kmdeck - refactoring to make code design better
- [x]  #611  - easy fix for consistency
- [ ] #661 @kmdeck
- [ ] callback ordering problem #1241 @imreddyTeja 
- [ ] roughness lengths #1450

**Comment by kmdeck** [comment]
In addition to any new "features" we want, we probably want to do some cleanup of old issues. I will go through and link issues here that I think we can and should close prior to release. New features and capabilities we want to make sure we have we can discuss in person!

**Comment by AlexisRenchon** [comment]
Thank you this is great!! 
I updated the initial comment of this issue, just copy-pasting your list, and added tick box.
